control-plane-operator/hostedclusterconfigoperator/controllers/resources/resources.go

```diff
@@ -277,6 +278,20 @@ func (r *reconciler) Reconcile(ctx context.Context, _ ctrl.Request) (ctrl.Result
 		errs = append(errs, fmt.Errorf("failed to reconcile kubernetes.default endpoints and endpointslice: %w", err))
 	}
 
+	releaseImageVersion, err := semver.Parse(releaseImage.Version())
+	if err != nil {
+		return ctrl.Result{}, fmt.Errorf("failed to parse release image version: %w", err)
+	}
+
+	// The exception for IBMCloudPlatform is due to the fact that the IBM will include new certificates for HCCO from 4.17 version
+	if !(hcp.Spec.Platform.Type == hyperv1.IBMCloudPlatform && (releaseImageVersion.Major == 4 && releaseImageVersion.Minor < 17)) {
```

Comment:
Hey @rtheis I think this is what you were mentioning, is that alright?
Regarding the DisablePKI annotation, yes, it's compatible.